CodingDojo/server/src/main/java/com/codenjoy/dojo/services/AutoSaver.java

```diff
@@ -34,7 +34,8 @@
 @RequiredArgsConstructor
 public class AutoSaver extends Suspendable implements Tickable {
 
-    public static final int TICKS = 30;
+    @Value("${game.save.ticks}")
+    private int ticks;
```

Comment:
Can we disable it by setting game.save.auto to false. Check following method:
@Value("${game.save.auto}")
    public void setActive(boolean active) {
        super.setActive(active);
    }